LDAR_Sim/src/dev_tools/benchmarking_parser.py

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        benchmarking_parser.py
+# Purpose:     Parser for benchmarking results
+#
+# Copyright (C) 2018-2021  Intelligent Methane Monitoring and Management System (IM3S) Group
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+import pstats
+import sys
+import os
+
+benchmark_res_path = "Benchmarking"
+
+file_list = os.listdir(benchmark_res_path)
+
+for filename in file_list:
+    if not filename.endswith(".py"):
```

Comment:
Instead of filtering out python files, maybe just have this find the benchmark file name - You can set that as a constant